### DISCUSSION TITLE
?

### DISCUSSION THREAD

**Discussion by hackmoi007** [aryainjas/Microllect#2] [General]
(moi㉿kali)-[~/Microllect]
└─$ sudo python microllect.py
[sudo] Mot de passe de moi : 
Traceback (most recent call last):
  File "/home/moi/Microllect/microllect.py", line 12, in <module>
    import PySimpleGUI as sg
ModuleNotFoundError: No module named 'PySimpleGUI'

**Comment by aryainjas** [comment]
pip install PySimpleGUI
